base/dates/conversions.jl

```diff
@@ -129,3 +117,16 @@ Takes the given `DateTime` and returns the number of Julian calendar days since
 epoch `-4713-11-24T12:00:00` as a `Float64`.
 """
 datetime2julian(dt::DateTime) = (value(dt) - JULIANEPOCH)/86400000.0
+
+function Base.convert(::Type{DateTime}, c::Libc.ComputerTime)
+    base = unix2datetime(c.seconds)
+    extra = Millisecond( round( c.microseconds / 1000 ) )
+    base + extra
+end
+
+function Base.string(c::Libc.ComputerTime)
+    string( convert( DateTime, c) ) * " UTC"
+end
+show
```

Comment:
`string( now() )` returns `2016-10-28T13:33:03.539 UTC`. What changes should I make?